### PR TITLE
Modify artifact retrieval to "browsertime-videos-" instead of "browse…

### PR DESCRIPTION
**Pull request by alexandru-io**
…rtime-results", and update to 0.2.8

### CODE REVIEW

File: mozperftest_tools/mozperftest_tools/side_by_side.py [hunk 1]
```diff
@@ -593,6 +596,13 @@ def run(self, test_name="", new_test_name="", platform="", new_platform="", vism
                 "WARNING: Downloads will not be skipped as you are using something other "
                 "than the similarity metric (only supported for this metric)."
             )
+        if original:
+            artifacts_to_get = ["browsertime-videos-original"]
+        else:
+            artifacts_to_get = [
+                "browsertime-videos-annotated",
```
Comment (gmierz) [inline mozperftest_tools/mozperftest_tools/side_by_side.py:603]: Why do you need to download both of these when `--original` isn't used?

Comment (alexandru-io) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: if `original=True` then only `original` artifact is used, otherwise both `original` and `annotated`
`original=False` by default.

Comment (alexandru-io) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: original: If set, use the original videos in the side-by-side instead of the postprocessed videos

Comment (gmierz) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: Ok, but why do we need to download both the annotated, and the original videos if we're only going to be processing the annotated ones?

Comment (alexandru-io) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: if `original` is `True` then use only `original`, else use both.

Comment (gmierz) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: Currently the `original` flag causes the code to use only the original videos, and doesn't touch the annotated ones when it's provided. With the changes here, you're downloading both the annotated, and original videos, but you're only processing the annotated ones.

Comment (alexandru-io) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: sparky, can I have the line that is processing only the annotated ones despite downloading both, please?

Comment (gmierz) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: https://github.com/mozilla/mozperftest-tools/blob/master/mozperftest_tools/mozperftest_tools/side_by_side.py#L697-L698

Comment (gmierz) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: Previously there was only one artifact to download regardless of the original flag, but now there's two artifacts and the one we choose to download depends on this flag.

Comment (alexandru-io) [reply mozperftest_tools/mozperftest_tools/side_by_side.py:603]: Got it, thanks!

File: mozperftest_tools/mozperftest_tools/side_by_side.py [hunk 2]
```diff
@@ -148,20 +148,23 @@ def generate_step_chart(self, oldvid, newvid, vismetPath, prefix, metric, output
         plt.savefig(str(output / "%s-%s-step.png" % (prefix.rstrip("_"), metric)))
         plt.clf()
 
-    def _search_for_paths(self, rev_ids, artifact, open_data=False):
+    def _search_for_paths(self, rev_ids, artifacts, open_data=False):
```
Comment (gmierz) [inline mozperftest_tools/mozperftest_tools/side_by_side.py:151]: Could we revert the changes to `_search_for_paths` given that we no longer need the multi-artifact functionality? The alternative is that we'll need to modify this to properly organize the artifacts we are gathering into a dictionary.

File: mozperftest_tools/mozperftest_tools/utils/artifact_downloader.py [hunk 3]
```diff
@@ -486,17 +486,17 @@ def _check_unzip(filen):
                             return
 
                     for artifact in artifacts:
-                        aname = _pattern_match(artifact["name"], artifact_to_get)
```
Comment (gmierz) [inline mozperftest_tools/mozperftest_tools/utils/artifact_downloader.py:489]: Much better naming, thanks :)

**Comment by gmierz** [comment]
@alexandru-io other than the comment about the search for paths function, everything else looks good!

**Review by gmierz — APPROVED** [review]
r+ thanks for fixing this @alexandru-io!
